WatchApp Extension/Scenes/GlucoseChartScene.swift

```diff
@@ -28,7 +28,7 @@ private enum NodePlane: Int {
 private extension SKLabelNode {
     static func basic(at position: CGPoint) -> SKLabelNode {
         let basic = SKLabelNode(text: "--")
-        basic.fontSize = 12
+        basic.fontSize = 15
```

Comment:
This is too big for 38mm

Reply:
instead of hard-coding the font size, use the system-provided size for .caption2
```
UIFont.preferredFont(forTextStyle: .caption2).pointSize
```